sessions/talks/ancient-greek-philosophy-medieval-mental-models-and-21st-century-technology.md

```diff
@@ -5,3 +5,12 @@ speaker: owen-campbell
 I'll talk about how the use of paper systems led to a simplification of the ideas developed by Aristotle over two thousand years ago and how the thinking that we encode into our software today is still limited by that simplification.
 
 I'll give some simple examples and then describe how, for the first time in human history, we now possess technology that is perfectly capable of implementing a far richer understanding of the world than Aristotle could possibly have imagined - but how we need to change our own mental models to take advantage of that possibility.
+
+# Fair Warning
```

Comment:
Could you make this an <h2>?  You'd do this by using `##` rather than `#`.  Other than that, looks great.